setup.py

```diff
@@ -87,7 +87,7 @@ def package_files(directory):
         "graphlib_backport; python_version < '3.9'",
         "jinja2",
         "matplotlib",
-        "numpy",
+        "numpy < 1.20",
```

Comment:
```
"numpy < 1.20; python_version < '3.7'"
"numpy; python_version >= '3.7'"
```
?

Reply:
No, as discussed on slack, pip does not anyway. And then we have the mypy failures again on > 3.6.